<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainView</class>
 <widget class="QWidget" name="MainView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: rgb(40, 40, 40);
	color: rgb(235, 219, 178);
}

QPushButton {
	border-color: rgb(29, 32, 33);
	border-width: 4px;
	border-style: solid;
}
</string>
  </property>
  <widget class="QLabel" name="lbl_time">
   <property name="geometry">
    <rect>
     <x>423</x>
     <y>10</y>
     <width>160</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);
</string>
   </property>
   <property name="text">
    <string>00:00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_date">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>50</y>
     <width>160</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(29, 32, 33);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>130</width>
     <height>561</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="sizeConstraint">
     <enum>QLayout::SetFixedSize</enum>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="btn_navToMain">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>128</horstretch>
        <verstretch>128</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>128</width>
        <height>128</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>128</width>
        <height>128</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>14</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>MESSAGE</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_navToDiag">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>128</horstretch>
        <verstretch>128</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>128</width>
        <height>128</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>0</width>
        <height>128</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>14</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>DIAGNOSTIC</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_navToStop">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>128</horstretch>
        <verstretch>128</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>128</width>
        <height>128</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>128</width>
        <height>128</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>14</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>SHIFT</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_navToConf">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>128</horstretch>
        <verstretch>128</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>128</width>
        <height>128</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>128</width>
        <height>128</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>14</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>CONFIG</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lbl_indicatorGps">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>10</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>NAV</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_indicatorSrv">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>10</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>SRV</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>label</zorder>
  <zorder>lbl_time</zorder>
  <zorder>lbl_date</zorder>
  <zorder>lbl_indicatorGps</zorder>
  <zorder>lbl_indicatorSrv</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
